### DISCUSSION TITLE
How to translate selected word?

### DISCUSSION THREAD

**Discussion by majdghithan** [laravel/nova-issues#5964] [Q&A] [answered]
I've tried to add it to translations json but it does not work

**Comment by majdghithan** [comment]
<img width="164" alt="Screenshot 2023-10-14 at 00 18 43" src="https://github.com/laravel/nova-issues/assets/54972678/7bec82f6-2da4-4b97-afbf-e42a5a17389e">

**Comment by milewski** [comment, marked as answer]
it works for me: 

![image](https://github.com/laravel/nova-issues/assets/2874967/c2b373e2-ec1e-4517-8b7a-e1cf4eefeb4b)


```json5
// lang/vendor/nova/pt.json
":amount selected": ":amount selecionado",
```

**Reply by majdghithan** [reply]
Thank you, I found out statikbe/laravel-nova-chained-translation-manager was causing the issue, I deleted the package and it worked
